### DISCUSSION TITLE
CLI Version

### DISCUSSION THREAD

**Discussion by BillBuilt** [golang-migrate/migrate#724] [Q&A]
Hello,

Probably a silly question but searching for `cli version` seems to only return results about migration versions, understandably, but I was trying to figure out how to determine what the version of the CLI is. Is this possible? I installed it via the via `go install -tags 'postgres mysql' github.com/golang-migrate/migrate/v4/cmd/migrate@latest`  and when I enter `migrate -version` it just says `dev`. How do I determine what CLI version I am using?

Been using this for a while but never really needed to check what CLI version I had until now.

Thanks!

**Comment by dhui** [comment]
It looks like you're installing [via the Go toolchain](https://github.com/golang-migrate/migrate/tree/master/cmd/migrate#with-go-toolchain). See the instructions on how to install via the Go toolchain with a version.
